src/__tests__/__snapshots__/Footer.js.snap

```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders Footer correctly 1`] = `
+<footer
+  className="footer"
+>
+  <div
+    className="container"
+  >
```

Comment:
- [ ] Great job adding test. please ensure that all your test are passing (green)
![Screenshot (505)](https://user-images.githubusercontent.com/65298048/194600033-ca72078f-b283-4943-81ef-437371445d8d.png)

Reply:
sorry to interrupt , but you have to press 'u' to update and then run the tests again :)